flutter_app/lib/modules/change_drone_mode.dart

```diff
@@ -15,5 +18,7 @@ class ChangeDroneMode {
     var frame = setMode(comm.sequence, systemID, componentID, baseMode);
     comm.sequence++;
     comm.write(frame);
+
+    log('${mavModes[baseMode]} mode set.');
```

Comment:
move the `mavModes` map here and then import it in the widget (it's the other way around now)